dissect/target/loaders/dir.py

```diff
@@ -17,9 +17,13 @@ class DirLoader(Loader):
 
     @staticmethod
     def detect(path: Path) -> bool:
+        if (path / "fs").exists():
```

Comment:
I think this check is a bit too assuming. Could you add some more conditions? E.g. only do this if `find_dirs` returns `None` first?